apps/e2e/indicators-storybook/src/app/tokens/tokens.module.ts

```diff
@@ -9,12 +8,7 @@ import { TokensComponent } from './tokens.component';
 const routes: Routes = [{ path: '', component: TokensComponent }];
 @NgModule({
   declarations: [TokensComponent],
-  imports: [
-    CommonModule,
-    NoopAnimationsModule,
```

Comment:
The individual modules for stories should not have been importing `NoopAnimationsModule` as it should only be imported once per app.